packages/default-theme/src/components/SwCartProduct.vue

```diff
@@ -38,10 +41,32 @@ export default {
     },
   },
   setup(props, { root }) {
+    const { apiInstance } = getApplicationContext(root, "SwCartProduct")
     const { removeProduct, changeProductQuantity } = useCart(root)
-
+    const productUrl = ref("")
     const quantity = ref(props.product.quantity)
     const productImage = computed(() => getProductMainImageUrl(props.product))
+    // it's not 1:1 to Product entity interface
+    const regularPrice = computed(
+      () =>
+        (props.product.price.listPrice &&
+          props.product.price.listPrice.price) ||
+        props.product.price.unitPrice
+    )
+    const specialPrice = computed(
+      () => props.product.price.listPrice && props.product.price.unitPrice
+    )
+
+    onMounted(async () => {
+      // async load of seoUrls only
+      const response = await getProduct(
```

Comment:
@mkucmus saw this by chance: wouldn't it be better to move this to SwCart, make a batch request (listing endpoint?) there and give the URL as a prop to SwCartProduct? Imagine having 10+ items in cart, this would mean lots of requests...

Reply:
@niklaswolf thanks for watching :muscle: Yeah, It would be way efficient. It's the draft so it's not too late for little enhancements ;)